README.md

```diff
@@ -40,6 +40,7 @@ OMERO.server configuration.
 
 OMERO system user, group, permissions, and data directory.
 You may need to change these for in-place imports.
+Directory paths must not have a trailing `/`
```

Comment:
"Unstable features" sounds like a good compromise and then we try to fix it up across the board later.